packages/mobile/src/navigator/types.tsx

```diff
@@ -236,13 +240,6 @@ export type StackParamList = {
     origin: SendOrigin
     forceCurrency?: Currency
   }
-  [Screens.SendAmountLegacy]: {
```

Comment:
Hmm, why are we deleting this?

Reply:
This was duplicated in the types file so I deleted the duplicate. The SendAmountLegacy is still present. https://github.com/valora-inc/wallet/blob/6cd83568c0c47f3aab6c761cba4489165469476a/packages/mobile/src/navigator/types.tsx#L236-L242